src/js/common/schemaform/save-load-actions.js

```diff
@@ -161,14 +170,14 @@ export function fetchInProgressForm(formId, migrations) {
     // If we don't have a userToken, fail safely
     if (!userToken) {
       dispatch(setFetchFormStatus(LOAD_STATUSES.noAuth)); // Shouldn't get here, but just in case
-      return;
+      return Promise.reject('no auth'); // Returns rejected Promise for testing only
```

Comment:
Oh that's true. I was thinking about that the other day but never wrote that down to circle back around to it. I'm thinking wrapping it in an `if (__BUILDTYPE___ === 'development)`. What say ye?